tasks/security_agent.py

```diff
@@ -29,6 +29,22 @@
 ]
 
 
+def get_go_env(ctx, go_version):
```

Comment:
Why is this needed?

Reply:
It is used by the `functional_tests` target. In a previous version, we modified all the targets where the logic was copy/pasted (around 5 times) to use this function. But we decided to keep this commit and submit it later, to make the changes on common parts as small as possible. Does it make sense ?